### PR TITLE
[post] Automated framework migration - part #1

### PR DESCRIPTION
**Pull request by TomasVotruba**
- [post] How to Migrate Legacy PHP Applications Without Stop Developing New Features
- [post] add image for automated_migration
- [post] fix newlines
- add metadata
- grammarly
- present tense, author link

### CODE REVIEW

File: data/blog/posts/2022/2022-02-01-how-to-migrate-legacy-php-applications-without-stopping-development-of-new-features.md [hunk 1]
```diff
@@ -7,32 +7,36 @@ perex: |
...
+
+We are free from conflicts by always editing the app code in the "Development branch" and only adding files in the migration branch. We can also make our "Migration branch" to the "Development branch" by merging them.
 
-The most important thing was to avoid conflicts.
-We were careful not to edit the same file in the “Development branch” and the “Migration branch”. We continued developing new features as usual in the “Development branch” directory, for
-example, `old/app`. Then we applied Rector rules to migrate `old/app` and copied the whole result to `new/app` in the “Migration branch”, which automated the process in CI.
-We were free from conflicts by always editing the app code in the “Development branch” and only adding files in the migration branch. We also could make our “Migration branch” to the “Development branch” by just merging them.
+<br>
```
Comment (TomasVotruba) [inline data/blog/posts/2022/2022-02-01-how-to-migrate-legacy-php-applications-without-stopping-development-of-new-features.md:30]: The past tense (e.g. "was") usually describes some experience that is way back in history.

On the other hand, the present tense ("is") is more interactive and can drag reader right now.  

I've updated few sentences to follow this technique. Let me know if it's ok :+1:

**Comment by TomasVotruba** [comment]
Please check the text if it's ok for you, we can :ship:  it :)

File: data/blog/posts/2022/2022-02-01-how-to-migrate-legacy-php-applications-without-stopping-development-of-new-features.md (hunk 1 above)
Comment (rajyan) [reply data/blog/posts/2022/2022-02-01-how-to-migrate-legacy-php-applications-without-stopping-development-of-new-features.md:30]: Thank you! The changes are totally fine.

**Comment by rajyan** [comment]
@TomasVotruba 

Thanks for the fixes! Looks good :+1:

**Pull request merged**